dictionaryController.js

```diff
@@ -0,0 +1,141 @@
+const moment = require("moment");
+const http = require('http')
+const Sequelize = require("sequelize");
+const {
+  sequelize,
+  Dictionary,
+  Language
+
+} = require("../models");
+const Op = Sequelize.Op;
+
+exports.createDictionary = async (req, res) => {
+  res.header("Access-Control-Allow-Origin", "*");
+  const { word, defn, type,example,language } = req.body;
+  try {
+    const request = await Dictionary.create({
+      word:word,
+      defn:defn,
+      type:type,
+      example:example,
+      language:language
+    });
+    
+    return res.json(request);
+  } catch (err) {
+    console.log(err);
```

Comment:
It would be better if a specific and detailed error message is returned to the client.